### PR TITLE
UI tweaks to the connect screen

### PR DESCRIPTION
**Pull request by devoncarew**
Various UI tweaks:
- increase padding between some text on the connect screen
- make the URL textfield on the connect screen auto-focus

### CODE REVIEW

File: packages/devtools_app/lib/src/flutter/connect_screen.dart [hunk 1]
```diff
@@ -63,6 +63,7 @@ class _ConnectScreenBodyState extends State<ConnectScreenBody> {
               'Connect to a Running App',
               style: textTheme.bodyText1,
             ),
+            const Padding(padding: EdgeInsets.only(top: 6.0)),
```
Comment (jacob314) [inline packages/devtools_app/lib/src/flutter/connect_screen.dart]: nit:  probably more idiomatic to wrap the Text with the padding if you want to visualize this as padding 
or
use
Container(height: 6)
if you just want a spacer.

File: packages/devtools_app/lib/src/flutter/connect_screen.dart [hunk 2]
```diff
@@ -93,9 +95,8 @@ class _ConnectScreenBodyState extends State<ConnectScreenBody> {
                 // of hard coding material colors.
                 borderSide: BorderSide(width: 0.5, color: Colors.grey),
               ),
-              hintText: 'http://127.0.0.1:12345/AUTH_CODE=/',
+              hintText: 'http://127.0.0.1:12345/auth_code=',
```
Comment (jacob314) [inline packages/devtools_app/lib/src/flutter/connect_screen.dart]: Fyi @jwren  is this consistent with how you are displaying the hint text for auth codes in Intellij?

Comment (jwren) [reply packages/devtools_app/lib/src/flutter/connect_screen.dart]: I made changes in ~Q4 to both this hint text and the hint text in the Dart Plugin for IntelliJ to be consistent (including the capitalization).  If we think it looks better with lower case auth_code, I'll change it on the Dart Plugin side.

Comment (devoncarew) [reply packages/devtools_app/lib/src/flutter/connect_screen.dart]: I think it does look better. Likely useful for this to be in sync with other tools (if pretty low-priority for things like case consistency).

File: packages/devtools_app/lib/src/flutter/connect_screen.dart (hunk 1 above)
Comment (devoncarew) [reply packages/devtools_app/lib/src/flutter/connect_screen.dart]: Thanks! Updated to use `const SizedBox(height: 6)`.

**Comment by devoncarew** [comment]
Note sure what this is `submit-queue — Cirrus CI failed on master` (from the CI checks). Will land w/ it broken and follow up with Flutter engprod.

**Pull request merged**